packages/web-components/fast-components/src/anchored-region/README.md

```diff
@@ -0,0 +1,4 @@
+# fast-anchored-region
+An *anchored region* is a container component which enables authors to create layouts where the contents of the anchored region can be positioned relative to another "anchor" element.
```

Comment:
As a component title, if that's what it is, should the **A**nchored region be capitalized as I've done in this sentence?  Ultimately, should we be looking to nail down documentation and writing requirements? For example, all component names are capitalized, all headers are .... ?  This would be a good starting point for all the documentation we'll have to bring into consistency for launch.

Reply:
I think this would be good to open an issue. I think the feedback here is good, but the primary intent was to clean up missing parts and pieces. I think a consistency scrub and conversation would be good to have; would you like to do an initial run through the specs that we currently have and the readmes?